subprojects/architecture-test/architecture-test.gradle.kts

```diff
@@ -6,6 +6,9 @@ plugins {
 }
 
 dependencies {
+    currentClasspath(project(":distributionsFull"))
+    currentClasspath(project(":pluginDevelopment"))
```

Comment:
Why is the ":pluginDevelopment" dependency needed?

Reply:
This one is transitively included in `:distributionsFull` via `gradle-kotlin-dsl-provider-plugins`. However, in this case `gradle-kotlin-dsl-provider-plugins` is not selected and thus all its transitives are lost as well. Perhaps `:pluginDevelopment` should be added as a direct dependency in `:distributionsFull`.

Reply:
Ah. Good catch. Yes, I think we should add it to `:distributionsFull` then.